openff/nagl_models/_dynamic_fetch.py

```diff
@@ -68,43 +72,39 @@ def get_model(
     Returns
     -------
     str
-        The path to the file if it was found. If the file wasn't found then a FileNotFoundError is rasied.
+        The path to the file if it was found. If the file wasn't found then a FileNotFoundError is raised.
 
     Raises
     ------
     HashComparisonFailedException
     FileNotFoundError
     """
-
+    if not(filename.endswith(".pt")):
+        raise BadFileSuffixError(f"NAGLToolkitWrapper does not recognize file path extension "
+                                 f"on {filename=}, expected '.pt' suffix")
```

Comment:
A little simpler to just refer to the models themselves

```suggestion
        raise BadFileSuffixError(f"OpenFF NAGL models are based on PyTorch files and expect a `.pt` suffix. Found an unrecognized file path extension "
                                 f"on {filename=}")
```